MultimodalQnA/docker_compose/intel/cpu/xeon/set_env.sh

```diff
@@ -6,30 +6,50 @@ pushd "../../../../../" > /dev/null
 source .set_env.sh
 popd > /dev/null
 
-export no_proxy=${your_no_proxy}
-export http_proxy=${your_http_proxy}
-export https_proxy=${your_http_proxy}
-export EMBEDDER_PORT=6006
-export MMEI_EMBEDDING_ENDPOINT="http://${host_ip}:$EMBEDDER_PORT/v1/encode"
-export MM_EMBEDDING_PORT_MICROSERVICE=6000
-export ASR_ENDPOINT=http://$host_ip:7066
+export host_ip=$(hostname -I | awk '{print $1}')
+export MM_EMBEDDING_SERVICE_HOST_IP=${host_ip}
+export MM_RETRIEVER_SERVICE_HOST_IP=${host_ip}
+export LVM_SERVICE_HOST_IP=${host_ip}
+export MEGA_SERVICE_HOST_IP=${host_ip}
+
+export no_proxy=${no_proxy}
+export http_proxy=${http_proxy}
+export https_proxy=${http_proxy}
+
+export WHISPER_PORT=7066
+export WHISPER_MODEL="base"
+export ASR_ENDPOINT=http://$host_ip:$WHISPER_PORT
+
+export ASR_PORT=9099
 export ASR_SERVICE_PORT=3001
 export ASR_SERVICE_ENDPOINT="http://${host_ip}:${ASR_SERVICE_PORT}/v1/audio/transcriptions"
-export REDIS_URL="redis://${host_ip}:6379"
+
+export REDIS_DB_PORT=6379
+export REDIS_INSIGHTS_PORT=8001
+export REDIS_URL="redis://${host_ip}:${REDIS_DB_PORT}"
 export REDIS_HOST=${host_ip}
 export INDEX_NAME="mm-rag-redis"
-export LLAVA_SERVER_PORT=8399
-export LVM_ENDPOINT="http://${host_ip}:8399"
+
+export DATAPREP_MMR_PORT=6007
+export DATAPREP_INGEST_SERVICE_ENDPOINT="http://${host_ip}:${DATRAPREP_MMR_PORT}/v1/ingest_with_text"
```

Comment:
All of these seem to have an extra `R`
```suggestion
export DATAPREP_INGEST_SERVICE_ENDPOINT="http://${host_ip}:${DATAPREP_MMR_PORT}/v1/ingest_with_text"
```

Reply:
oh no I thought I fixed that. Good catch thanks